specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -0,0 +1,5772 @@
+          },
+          {
+            "name": "metricName",
+            "in": "path",
+            "description": "The metric name",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "groupName",
+            "in": "path",
+            "description": "The group name",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "groupKey",
+            "in": "query",
+            "description": "The group key",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "fromDate",
+            "in": "query",
+            "description": "The start date.",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "toDate",
+            "in": "query",
+            "description": "The end date.",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/MetricSets"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/replicationstatus": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets Replication status for a given Active Directory Domain Service, that is onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServices_getReplicationStatus": {
+            "$ref": "./examples/ReplicationStatus.json"
+          }
+        },
+        "operationId": "addsServices_getReplicationStatus",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/ReplicationStatus"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/replicationsummary": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets complete domain controller list along with replication details for a given Active Directory Domain Service, that is onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServices_getReplicationSummary": {
+            "$ref": "./examples/ReplicationSummary.json"
+          }
+        },
+        "operationId": "addsServices_getReplicationSummary",

```

Comment:
Should this be listReplicationSummary and use x-ms-pageable?

Reply:
You are right.Changing it.